tests/abi/abi.c

```diff
@@ -0,0 +1,100 @@
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "for_each.h"
+#include "functions.h"
+
+#ifdef __x86_64__
+# include "platform_specific/x86_64.h"
+#elif __x86__
+# include "platform_specific/x86.h"
+#else
+ _Static_assert(0, "unsupported platform architecture");
+#endif
+
+/* LFSR generator */
+
+static uint64_t LFSR = 0xdc984ef2647af904ull;
```

Comment:
Is this a backdoor? Document how you obtained this value!!!

Reply:
Wait, the starting point matters? O_o I was acting under the impression that this is pretty much just a seed, so a I picked an arbitrary value.

Reply:
I'm joking, but in crypto this is actually an issue:
* https://en.wikipedia.org/wiki/Nothing-up-my-sleeve_number
* https://safecurves.cr.yp.to/rigid.html

Reply:
Ah, good, because you actually scared me for a second :D
I was frantically starting to google around whether my understanding of anything and everything is incomplete!